examples/framesjs-starter/app/examples/simplified-api/[[...routes]]/route.tsx

```diff
@@ -0,0 +1,72 @@
+import { AnyJson } from "../new-api";
+import {
+  FrameImage,
+  Frame,
+  FramesApp,
+  FrameLink,
+  renderFramesToResponse,
+} from "../react-api";
+
+type FrameState = {
+  clicked: number;
+};
+
+function isFrameState(
```

Comment:
if user needs type safety they should define the type or zod validator and the lib should assert the correct types rather than have the user implement a type validation function like this

Reply:
Yep those are possible solutions. Zod validator is the best I think as it actually validates.